Add OffBoarding test

Danielius1922 opened the pull request.

Danielius1922 committed:
Add OffBoarding tests

jkralik commented on test/cloud-server/Dockerfile:
```diff
@@ -7,15 +7,20 @@ COPY . .
+#certificate-generator
+ARG service=kit
+WORKDIR /
+RUN cd $GOPATH/pkg/mod/github.com/plgd-dev/kit/v2* && go build -ldflags "-linkmode external -extldflags -static" -o /go/bin/certificate-generator ./cmd/certificate-generator
```
use `tools/cert-tool/cmd`

Danielius1922 replied:
updated in Dockerfile.test, bundle/Dockerfile and test/cloud-server/Dockerfile

Danielius1922 committed:
Update Dockerfiles to use cert-tool

Use tools/cert-tool utility instead of certificate-generator
utilility from plgd-dev/kit/cmd/certificate-generator.
